glhe/aggregation/dynamic.py

```diff
@@ -111,22 +105,20 @@ def calc_temporal_superposition(self, time_step: int, flow_rate: float = None) -
             if not flow_rate:
                 g_b = self.interp_g_b.interpolate(lntts)
             else:
-                g_b = np.flipud(self.interp_g_b(lntts, flow_rate))
+                # TODO: This is not covered by tests, and may need adjusting the interpolator class to work properly
+                g_b = np.flipud(self.interp_g_b.interpolate(lntts, flow_rate))
```

Comment:
Still leaving this for now.  I think the best thing would be to work up a unit test that exercises it, see it fail, and then fix it :)